handsontable/src/selection/highlight/types/focus.js

```diff
@@ -7,11 +7,15 @@ import VisualSelection from '../visualSelection';
  *
  * @param {object} highlightParams A configuration object to create a highlight.
  * @param {Function} highlightParams.cellCornerVisible Function to determine if cell's corner should be visible.
+ * @param {Function} highlightParams.cellAttributes Array of cell attributes.
+ * @param {Function} highlightParams.headerAttributes Array of header attributes.
  * @returns {Selection}
  */
-export function createHighlight({ cellCornerVisible, ...restOptions }) {
+export function createHighlight({ cellCornerVisible, cellAttributes, headerAttributes, ...restOptions }) {
   return new VisualSelection({
     className: 'current',
+    cellAttributes,
+    focusedHeaderAttributes: headerAttributes,
```

Comment:
```suggestion
    headerAttributes: headerAttributes,
```
I'd rename the property to a more generic one. The `VisualSelection` class represents a variety of selection layers, and there should not be an option that matches the one layer - in this case "focus" layer.